apps/namadillo/src/App/Transfer/TransferDestination.tsx

```diff
@@ -47,6 +47,8 @@ export const TransferDestination = ({
   openChainSelector,
   openProviderSelector,
 }: TransferDestinationProps): JSX.Element => {
+  const isExternalDestination = !!openChainSelector;
```

Comment:
I'm not sure this is the correct way of verifying that we can define the fee token. I think we should verify if the transfer is coming from Namada instead

Reply:
yeah, agreed. I added the prop `changeFeeEnabled` to have a better control over it 👍